data/assets/scene/solarsystem/planets/jupiter/minor/ananke_group.asset

```diff
@@ -207,22 +207,61 @@ local anankeGroup = {
   }
 }
 
+--Generate all moon labels
+local moon_labels = {}
+
+for i, moon in ipairs(anankeGroup) do
+  moon_labels[i] = {
+    Identifier = moon.Identifier .. "Label",
+    Parent = moon.Identifier,
+    Renderable = {
+      Enabled = false,
+      Type = "RenderableLabel",
+      Text = moon.Identifier,
+      FontSize = 70.0,
+      Size = 7.1,
+      MinMaxSize = { 1, 25 },
+      OrientationOption = "Camera View Direction",
+      BlendMode = "Additive",
+      EnableFading = true,
+      FadeUnit = "Gm",
+      FadeDistances = { 1.5, 90.0 },
+      FadeWidths = { 0.5, 90.0 }
+    },
+    Tag = { "solarsystem_labels", "moon_labels", "minor_moon_labels" },
+    GUI = {
+      Name = moon.Identifier .. " Label",
+      Path = "/Solar System/Planets/Jupiter/Moons",
+      Description = "Label for Jupiter's moon " .. moon.Identifier .. " (Ananke group)"
+    }
+  }
+end
+
+
 local nodes = proceduralGlobes.createGlobes(anankeGroup)
 
 asset.onInitialize(function()
   for i, node in ipairs(nodes) do
     openspace.addSceneGraphNode(node)
   end
+
+  for i, moonlabel in ipairs(moon_labels) do
```

Comment:
The Lua-esque way of dealing with the parameter you don't want is to use `_` instead [which should have happened in the list above as well], so:
```lua
for _, moonLabel in ipairs(moon_labels) do
```

Reply:
Same for a bunch of other places

Reply:
Fixed in: [055eab06540015eba3138c73cb4cb4bf6f36b5ad](https://github.com/OpenSpace/OpenSpace/commit/055eab06540015eba3138c73cb4cb4bf6f36b5ad)